src/containers/pages/PlanAssignment/index.tsx

```diff
@@ -248,18 +248,18 @@ const PlanAssignment = (props: PlanAssignmentProps) => {
   /**
    * Check if map should be visible based on plan type
    */
-  const isMapEnabled = (): boolean => {
+  const isMapDisabled = (): boolean => {
     let interventionType: string | undefined = '';
     if (plan) {
       const type = plan.useContext.find(element => element.code === INTERVENTION_TYPE_CODE);
       type ? (interventionType = type.valueCodableConcept) : (interventionType = '');
     }
-    return MAP_ENABLED_PLAN_TYPES.includes(interventionType);
+    return MAP_DISABLED_PLAN_TYPES.includes(interventionType);
   };
 
   return (
     <>
-      {isMapEnabled ? <ConnectedAssignmentMapWrapper {...AssignmentWraperProps} /> : ''}
+      {isMapDisabled ? '' : <ConnectedAssignmentMapWrapper {...AssignmentWraperProps} />}
```

Comment:
Should we maybe change this to `!isMapDisabled && <ConnectedAssignmentMapWrapper {...AssignmentWraperProps} />`?  This is so that we avoid the weird empty string output